dts/bindings/clock/silabs,series2-lfrco.yaml

```diff
@@ -0,0 +1,13 @@
+compatible: "silabs,series2-lfrco"
+
+description: |
+  Silicon Labs LFRCO peripheral.
```

Comment:
Help messages are the right place to expand acronyms (in many case it provide a sufficient documentation).